app/models/hearing_summary.rb

```diff
@@ -43,11 +43,21 @@ def court_centre
     HmctsCommonPlatform::CourtCentre.new(body["courtCentre"])
   end
 
+  def defence_counsel_ids
+    defence_counsels.map(&:id)
+  end
+
 private
 
   def hearing_days
     Array(body["hearingDays"]).map do |hearing_day_data|
       HmctsCommonPlatform::HearingDay.new(hearing_day_data)
     end
   end
+
+  def defence_counsels
+    Array(body["defenceCounsel"]).map do |defence_counsel_data|
```

Comment:
Do you have a proper updated schema? I'm only going off the diagram in the solution paper, but it says there's 0 or 1 defence counsel on there

Reply:
the `0..1` is near `HearingSummary`, and the `DefenceCousel` has a `*`. Not entirely sure what that means, but I interpret it as `0 or 1 hearing summary has many DefenceCounsel`.

Reply:
Right I see that now. Their way of allowing the diagram to show if it's a one to many or many to many relationship I guess